config/routes/candidate.rb

```diff
@@ -580,5 +580,7 @@
 
   get '/account-locked', to: 'errors#account_locked'
 
+  get '/guidance', to: 'guidance#index'
```

Comment:
@MaeveRoseveare @JR-G any thoughts what the URL should be? `/candidate/guidance` is pretty generic (and we might need to add other guidance pages in future?) but maybe it's ok?

Other options:

* `/candidates/about-the-applications-process`
* `/candidates/applications-process`
* `/candidates/how-applications-work`
* ???

It matters a bit, as hopefully this URL will get emailed out to candidates from teacher training advisers and perhaps in other emails - plus hopefully it'll appear in Google search results.

Reply:
I like `/candidates/about-the-applications-process`!

Reply:
I agree with James